src/ResourceManager/Sql/Commands.Sql/Elastic Pools/Model/AzureSqlElasticPoolModel.cs

```diff
@@ -78,6 +84,28 @@ public class AzureSqlElasticPoolModel
         /// </summary>
         public int? DatabaseDtuMin { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Sku capacity of the elastic pool
+        /// </summary>
+        public int? Capacity { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Minumum capacity of the elastic pool
+        ///    For Dtu pool, it is the min Database Dtu of the pool; for Vcore pool, it is the min vcore capacity of the pool
+        /// </summary>
+        public double? MinCapacity { get; set; }
```

Comment:
It seems like for consistency these should be named DatabaseCapacityMin and DatabaseCapacityMax

Reply:
@jaredmoo Yes, rename both.